chain-signatures/node/src/protocol/cryptography.rs

```diff
@@ -427,8 +427,6 @@ impl CryptographicProtocol for RunningState {
         // block height is up to date, such that they too can process signature requests. If they cannot
         // then they are considered unstable and should not be a part of signature generation this round.
         let stable = ctx.mesh().stable_participants().await;
-        tracing::info!(?stable, "stable participants");
```

Comment:
Do we have it logged somewhere else?
If its spammy, we can lower the log level to trace.

Reply:
added back as trace